crates/language/src/target_language.rs

```diff
@@ -245,6 +288,7 @@ impl PatternLanguage {
     }
 
     pub fn from_extension(extension: &str) -> Option<Self> {
+        Self::from_string_or_alias(extension, None)
```

Comment:
This call isn't doing anything, I think you probably meant to delegate this entirely?

Reply:
I have changed  the from_extension function which now uses the updated from_string function, which  handles both full language names and aliases.